### ISSUE TITLE
Reasonable plan and ideas for methods

### ISSUE THREAD

**Issue by WhimZig** [epfl-ada/ada-2023-project-amonavis#8]
This is the part that I'm less sure about how to fill in. Especially when it comes to mind of adding the essential mathematical details in the notebook. So...

I guess for this part just adding some alternative methods, or showing that the initial versions of the methods can be done is going to be okay? I'm really unsure of how to finish this bullet point tbh.

**Comment by DanielX278** [comment]
I guess with "essential" they just wanna be sure we know what we are doing and why (ie. we mention the specific "cosine similarity" so that they know we know what math foundations we are going to use). I updated the draft of the proposal, I mentioned a bunch of specific metrics we can use for the graph and something else, maybe I can try to be more specific? (like mentioning "yes we normalize the embedding on -1, +1" or stuff like that)

Also, I worked on your suggestion about the AI element in the project, and added some details here and there. Are they what you were thinking already? I also added elements in italics for the part i'm not really sure of, lmk if they make sense
